mail/src/main/java/com/sun/mail/imap/protocol/UIDSet.java

```diff
@@ -86,7 +86,6 @@ public long size() {
 	    return null;
 	List<UIDSet> v = new ArrayList<>();
 	StringTokenizer st = new StringTokenizer(uids, ",:", true);
-	long start = -1;
```

Comment:
Unused in commit introduced in https://github.com/javaee/javamail/commit/2bc042e76d63147093c80189c0d9aeab8c49f789